### ISSUE TITLE
Make print statements optional 

### ISSUE THREAD

**Issue by niekdejonge** [matchms/matchms#270]
Request:
Make the print statements in derive_adduct_from_name() optional. 

Current behaviour:
The function derive_adduct_from_name() prints print(f"Removed adduct {adduct_from_name} from compound name.") and         print(f"Added adduct {adduct_cleaned} to metadata.") for each spectrum. When running on a large dataset (in my case all GNPS spectra > 400.000 spectra), this results in a massive number of print statements. When run in jupyter notebook, this makes it very slow to later load in the jupyter notebook again. 

Improved behaviour:
Add a variable to the function to make printing statements optional.

**Comment by florian-huber** [comment]
I agree that this should be optional.
It also fits to another issues: #263 

Not to forget the general theme --> logging! #71

**Comment by florian-huber** [comment]
This will be solved with #271

**Comment by florian-huber** [comment]
#271 now contains the option to set the level of logging, e.g. to "INFO" (also containing information as mentioned above) or "WARNING" (only warnings and higher) or even highter to "ERROR" to silence all warnings. This can be done using:
```python
from matchms.logging_functions import set_matchms_logger_level

set_matchms_logger_level("INFO")
```

An alternative is to silence the ouput stream and only write it to a logging file:
```python
from matchms.logging_functions import add_logging_to_file

add_logging_to_file("sample.log", loglevel="INFO", remove_stream_handlers=True)
```
